pkg/action/diff.go

```diff
@@ -25,15 +26,15 @@ func relFileReport(ctx context.Context, c bincapz.Config, fromPath string) (map[
 		return nil, err
 	}
 	fromRelPath := map[string]*bincapz.FileReport{}
-	for _, f := range fromReport.Files {
-		if f.Skipped != "" || f.Error != "" {
+	for files := fromReport.Files.Oldest(); files != nil; files = files.Next() {
```

Comment:
This is the general pattern for `orderedmap`s -- `var.Key` and `var.Value`. Value fields can be accessed via `var.Value.Field`.